<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>B588</class>
 <widget class="QWidget" name="B588">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QCheckBox" name="cb_ignore_bad_cmd">
       <property name="text">
        <string>ignore bad cmd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_find_devices">
       <property name="text">
        <string>Найти устройства</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_address"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="grapher_layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
